front/packages/ui/src/browse/index.tsx

```diff
@@ -52,27 +52,30 @@ export const itemMap = (
 		item.kind === "show" ? item.watchStatus?.unseenEpisodesCount ?? item.episodesCount! : null,
 });
 
-const query = (sortKey?: SortBy, sortOrd?: SortOrd): QueryIdentifier<LibraryItem> => ({
+const query = (sortKey?: SortBy, sortOrd?: SortOrd, mediaTypeKey?: string): QueryIdentifier<LibraryItem> => ({
```

Comment:
Is there a reason you use strings instead of enum values?

Reply:
Good point, I've cleaned this up so it uses an enum, but also I've adjusted it so that once we parse the `mediaTypeParam` string, we get a `MediaType` object which is never undefined. I think this makes it easier to reason about and simplifies the code.